packages/conf-rust-wasm/conf-rust-wasm.1/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "Léo Andrès <contact@ndrs.fr>"
+authors: "Léo Andrès <contact@ndrs.fr>"
+homepage: "https://github.com/ocaml/opam-repository"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "MIT"
+build: [
+  ["sh" "-c" "rustc --print target-list | grep -q 'wasm32-unknown-unknown'" ]
+]
+depends: [
+  "conf-rust" {build}
+]
+depexts: [
+  ["rust-wasm"] { os-family = "alpine" }
+  ["rust-wasm"] { os-family = "arch" }
+  ["libstd-rust-dev-wasm32"] { os-family = "debian" }
+  ["libstd-rust-dev-wasm32"] { os-family = "ubuntu" }
+
+  ["libstd-rust-dev-wasm32"] { os-distribution = "centos" }
+  ["rust-std-static-wasm32-unknown-unknown"] { os-distribution = "fedora" }
+]
+available: os-family != "opensuse" & os-family != "suse" & os-distribution != "homebrew"
```

Comment:
I did not find how to have it on macOS, it seems weird but maybe it's just really lacking behind...

Reply:
```suggestion
```